desktop/mac/release-notes/index.md

```diff
@@ -17,14 +17,45 @@ redirect_from:
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
-## Docker Desktop 4.1.1
-2021-10-12
+## Docker Desktop 4.2.0
+2021-11-xx
 
 > Download Docker Desktop
 >
 > [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
 > [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
 
+### New
```

Comment:
nit: can you add a blank line after? (mostly for consistency; I think it usually works without, but we had some cases where it caused rendering issues)

```suggestion
### New

```